docs/README.md

```diff
@@ -0,0 +1,328 @@
+
+### Model danych
+
+Model danych jest dostępny w dokumentacji API.
+
+Wyróżniamy następujące elementy:
+
+* Dictionary - dane na temat słowników danych statystycznych
+* Case - dane na temat sprawy
+* Institution - informacje o instytucjach
+* Tag - dodatkowa konfiguracja prezentacji znaczników
+* Collection - kolekcje umożliwiające dokonanie wyboru prezentowanych spraw poprzez wskazanie kryterium ich wyszukiwania
+* Letter - dane na temat listów przypisanych do sprawy / nowych
+* File - dane na temat plików powiązanych z listem
+* Channel - słownik zawierający konfiguracje kanałów komunikacji
+* Note - notatka wprowadzona przez użytkownika
+* Event - dane na temat wydarzenia terminarza
+* User - dane na temat użytkowników systemu
+* Description - słownik opisów listów
+* AdministrativeUnit - jednostki podziału terytorialnego
+
+### Model uprawnień
+
+System wyróżnia następujące uprawnienia:
+
+* Dictionary.change
+* Dictionary.view
+* Dictionary.delete
+* Dictionary.create
+* Case.change
+* Case.view
+* Case.delete
+* Case.create
+* Institution.change
+* Institution.view
+* Institution.delete
+* Institution.create
+* Tag.change
+* Tag.view
+* Tag.delete
+* Tag.create
+* Collection.change_own
+* Collection.change_all
+* Collection.view
+* Collection.delete
+* Collection.create
+* Letter.change_own
+* Letter.change_all
+* Letter.view
+* Letter.delete
+* Letter.create
+* Channel.change
+* Channel.view
+* Channel.delete
+* Channel.create
+* Note.change_own
+* Note.change_all
+* Note.view
+* Note.delete
+* Note.create
+* Event.change_own
+* Event.change_all
+* Event.view
+* Event.delete
+* Event.create
+* User.change_own
+* User.change_all
+* User.view
+* User.delete
+* User.create
+* Description.change
+* Description.view
+* Description.delete
+* Description.create
+* AdministrativeUnit.change
+* AdministrativeUnit.view
+* AdministrativeUnit.delete
+* AdministrativeUnit.create
+
+Klasy uprawnień mają następujące znaczenie:
+
+* change - edycja wszystkich elementów danego typu
+* view - wyświetlanie wszystkich elementów danego typu
+* delete - usunięcie wszystkich elementów danego typu
+* change_own - edycja elementów danego typu utworzonych przez tego samego użytkownika
+* change_all - edycja elementów danego typu utworzonych przez dowolnego użytkownika
+
+W celu kontroli dostępu wykorzystywane jest:
+
+* uwierzytelnianie  poprzez GSuite - oznaczone w dokumentacji API jako ```sessionAuth```
+* uwierzytelnianie hasłem - wyłącznie w celach administracyjnych, tożsame z oznaczeniem w API jako ```sessionAuth```
+* uwierzytelnianie kodem JWT - oznaczone w dokumentacji API jako ```bearerAuth```.
+
+Dokumentacja API pomija uwierzytelnianie w ten sposób jako standardowy protokół.
+
+### Model powiadomień
+
+Model powiadomień zostanie zdefiniowany po zrealizowaniu podstawowych elementów edycji.
+
+## Panel Zarządzania
+
+### Założenia
+
+* framework - Vue/React
+* protokół komunikacji - REST
+* framework UI - Bootstrap / AntDesign / inny?
```

Comment:
>> "ustalenie podstawowego stylu i kształtu UI"
> 
> Przykładowe gotowe biblioteki:
> Material design dla Vue: https://vuematerial.io/components/app (ma inputa do plików, ale bez drag & drop)
> Material design dla Reacta: https://material-ui.com/getting-started/installation/ (istnieje pasujący gotowy komponent do uploadu plików z drag & drop: material-ui-dropzone)

Reply:
Osobiście preferuje Vue, gdyż mam z nim doświadczenie. Wiem jednak, że nie będę miał możliwości znaczącego zaangażowania w projekt, więc pozostawiam to do dyspozycji wykonawcy. Myślę, że obie technologie długofalowo są możliwe w utrzymaniu, a to jest dla mnie istotne.

Osobiście nie posiadam żadnego preferencji z zakresie frameworka UI. Uważam, że koniecznie musimy po taki sięgnąć. Do wyboru wykonawcy pozostawiam czy będzie to Material-Design, Ant-Design, czy inny, tak długo, jeżeli rozwiązanie będzie zapewniało długofalowe utrzymanie w czasie, czyli będzie aktywnie wykorzystywane przez istotne. Komponent do uploadu plików nie jest egzotyczny.

UI (Panel Zarządzania) jest wyłącznie elementem warstwy prezentacji, który stosunkowo prosto można zmienić, gdy istnieje API, zwłaszcza przy obecnych nie wygórowanych wymaganiach. Co innego zmiana struktury danych, albo - po opracowaniu innych integracji – zmiana struktury API.

Pozostawiam kwestie do dyspozycji wykonawcy także z tego powodu, aby nie zamykać projektu i ograniczać ochotników. Nie ma powodów wprowadzać nadmiernych wymagań. Zastanawiam się nawet – aby zwiększyć szanse realizacji – w jakich technologiach odpowiednich do wykoniania Content-API ma doświadczenie SII, bowiem w ostatnim roku mocno skierowałem się w kierunku NodeJS / MongoDB, co poszerza możliwe technologie w jakich w przyszłości byłbym w stanie ewentualnie utrzymać projekt (niestety, realia są takie, że Stowarzyszenie nie ma obecnie innej osoby, która utrzymywałaby projekty Stowarzyszenia, a mój czas na adaptacje jest ograniczony).